### PR TITLE
Add information to diagnostics bundle

### PR DESCRIPTION
**Pull request by janisz**
This PR adds folowing information to diagnostics bundle

* [`/quota`](http://mesos.apache.org/documentation/latest/endpoints/master/quota/)
* [`/containers`](http://mesos.apache.org/documentation/latest/endpoints/slave/containers/)
* [`/proc/[cmdline|cpuinfo|meminfo]`](http://man7.org/linux/man-pages/man5/proc.5.html) – additional hardware infro
* [`ps aux www`](http://man7.org/linux/man-pages/man1/ps.1.html) – lists all proceses with wide information about them

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -511,17 +511,23 @@ def _download_bundle_from_master(dcos_api_session, master_index):
     expected_common_files = ['dmesg-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
                              'opt/mesosphere/etc/dcos-version.json.gz', 'opt/mesosphere/etc/expanded.config.json.gz',
                              'opt/mesosphere/etc/user.config.yaml.gz', 'dcos-diagnostics-health.json',
-                             'var/lib/dcos/cluster-id.gz']
+                             'var/lib/dcos/cluster-id.gz', 'ps_aux_ww.gz', '/proc/cmdline', '/proc/cpuinfo',
```
Comment (klueska) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: The files currently called `'/proc/cpuinfo'` etc. should be `'proc/cpuinfo.gz'`, etc.

Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Fixed. Test passed because `/proc/cpuinfo` is existing file on the system

**Comment by janisz** [comment]
@surdy @klueska @philipnrmn Can you take a look at this? I think it's ready to merge

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by klueska** [comment]
@mesosphere-mergebot label "Ready For Review"

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 2]
```diff
@@ -511,17 +511,23 @@ def _download_bundle_from_master(dcos_api_session, master_index):
...
                              'opt/mesosphere/etc/dcos-version.json.gz', 'opt/mesosphere/etc/expanded.config.json.gz',
                              'opt/mesosphere/etc/user.config.yaml.gz', 'dcos-diagnostics-health.json',
-                             'var/lib/dcos/cluster-id.gz']
+                             'var/lib/dcos/cluster-id.gz', 'ps_aux_ww.gz', 'proc/cmdline.gz', 'proc/cpuinfo.gz',
+                             'proc/meminfo.gz']
 
     # these files are expected to be in archive for a master host
     expected_master_files = ['dcos-mesos-master.service.gz', 'var/lib/dcos/exhibitor/zookeeper/snapshot/myid.gz',
-                             'var/lib/dcos/exhibitor/conf/zoo.cfg.gz'] + expected_common_files
+                             'var/lib/dcos/exhibitor/conf/zoo.cfg.gz', '5050-quota.json'
```
[the remark this answers is not in the document]
Comment (surdy) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: For some reason files generated by the HTTPEndpoints section were not checked for in the integration tests. I lazily never dug into why. Does someone from Cluster Ops know why ?

[the remark this answers is not in the document]
Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Any idea @klueska

**Comment by janisz** [comment]
@GoelDeepak @gpaul  Can you look at this?

File: gen/dcos-config.yaml [hunk 3]
```diff
@@ -836,6 +841,11 @@ package:
                   "Port": 5051,
                   "Uri": "/overlay-agent/overlay",
                   "Role": ["agent", "agent_public"]
+              },
+              {
+                  "Port": 5051,
+                  "Uri": "/containers",
+                  "Role": ["agent", "agent_public"]
               }
```
Comment (philipnrmn) [inline gen/dcos-config.yaml]: What info do we get from `containers` that we didn't have in `state`?

Comment (surdy) [reply gen/dcos-config.yaml]: Good question @philipnrmn . I'm not sure about @janisz's motivation, but I have used containers endpoint to get executor_pid in the past which I needed to kill a stuck task.

Comment (janisz) [reply gen/dcos-config.yaml]: It was requested by supporting team. I'm not sure why.

**Comment by jieyu** [comment]
@janisz can we also add `/proc/self/mountinfo`?

**Comment by janisz** [comment]
@jieyu Sure, i created an issue for this https://jira.mesosphere.com/browse/DCOS_OSS-3961

**Pull request merged**